### PR TITLE
Add DB Ready Check

### PR DESCRIPTION
**Pull request by RuliXu**
Fix:https://github.com/clusterpedia-io/clusterpedia/issues/85

**Before**
```
[root@master1 ~]# kubectl get po -n clusterpedia-system
NAME                                                                                READY   STATUS    RESTARTS      AGE
clusterpedia-apiserver-7787484686-wwdb8                        1/1     Running    3 (61s ago)      109s
clusterpedia-clustersynchro-manager-56777bf9ff-swbn5   1/1     Running    3 (73s ago)      109s
clusterpedia-postgresql-0                                                    1/1     Running     0                     109s
[root@node1 ~]# kubectl get po -n clusterpedia-system
NAME                                                                            READY   STATUS    RESTARTS      AGE
clusterpedia-apiserver-7787484686-7s9m7                     1/1     Running     4 (73s ago)     2m3s
clusterpedia-clustersynchro-manager-56777bf9ff-jzplv   1/1     Running     4 (76s ago)     2m3s
clusterpedia-mysql-0                                                        1/1     Running     0                     2m3s
```

**After**
```
[root@master1 ~]# kubectl get po -n clusterpedia-system
NAME                                                                                     READY   STATUS    RESTARTS   AGE
clusterpedia-apiserver-7848fd7c86-4lbdd                             1/1         Running    0                   61s
clusterpedia-clustersynchro-manager-6959945447-p7bcg   1/1         Running    0                   61s
clusterpedia-postgresql-0                                                       1/1         Running    0                   61s
[root@node1 ~]# kubectl get po -n clusterpedia-system
NAME                                                                                  READY   STATUS    RESTARTS   AGE
clusterpedia-apiserver-85fb97c48d-9qn6z                         1/1         Running    0          67s
clusterpedia-clustersynchro-manager-77957d57f-6zzn7   1/1         Running    0          67s
clusterpedia-mysql-0                                                          1/1         Running    0          67s
```

Signed-off-by: RuliXu <lily.xu@daocloud.io>

### CODE REVIEW

**Comment by calvin0327** [comment]
@Iceber /cc

File: charts/templates/clustersynchro-manager-deployment.yaml [hunk 1]
```diff
@@ -29,6 +29,25 @@ spec:
         {{- end }}
     spec:
       {{- include "clusterpedia.clustersynchroManager.imagePullSecrets" . | nindent 6 }}
+      initContainers:
+      - name: init-storage
```
Comment (Iceber) [inline charts/templates/clustersynchro-manager-deployment.yaml:33]: `check-storage` is better?

Comment (Iceber) [inline charts/templates/clustersynchro-manager-deployment.yaml:33]: `check-storage` is better?

**Review by Iceber — APPROVED** [review]
LGTM

**Pull request merged**